eZ/Publish/Core/MVC/Symfony/Security/UserChecker.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\Security;
+
+use eZ\Publish\API\Repository\UserService;
+use Symfony\Component\Security\Core\Exception\CredentialsExpiredException;
+use Symfony\Component\Security\Core\Exception\DisabledException;
+use Symfony\Component\Security\Core\User\UserCheckerInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+use eZ\Publish\Core\MVC\Symfony\Security\UserInterface as EzUserInterface;
+
+final class UserChecker implements UserCheckerInterface
```

Comment:
User Checker needs to be enabled in the firewall configuration. Ref. https://symfony.com/doc/current/security/user_checkers.html#enabling-the-custom-user-checker

Reply:
Yep. I can do it now, or after this PR is approved, you can choose :)

Reply:
I'm currently at a conference, so give me an hour or two :)

Reply:
No problem, take your time ;-)

Reply:
PR @ https://github.com/ezsystems/ezplatform/pull/469

Reply:
Thnak you ;-)